net._86box._86Box.yaml

```diff
@@ -1,6 +1,6 @@
 app-id: net._86box._86Box
 runtime: org.kde.Platform
-runtime-version: 5.15-23.08
+runtime-version: 6.6
```

Comment:
By the way, why using the 6.6 runtime instead of 6.7?

Reply:
The runtime is 6.8 now